lib/o365_mgmnt/index.js

```diff
@@ -10,49 +10,15 @@
 
 const util = require('util');
 
-const m_alUtil = require('../al_util');
 const m_o365mgmnt = require('./o365management');
 
-const msRest = require('ms-rest');
-const msRestAzure = require('ms-rest-azure');
-const msWebResource = msRest.WebResource;
+const {ApplicationTokenCredentials} = require('@azure/ms-rest-nodeauth');
 
-var fileTokenCache = require('azure/lib/util/fileTokenCache');
-var tokenCache = new fileTokenCache(m_alUtil.getADCacheFilename(
-    { 
-        creds : {
-            client_id : process.env.CUSTOMCONNSTR_APP_CLIENT_ID,
-            tenant_id : process.env.APP_TENANT_ID
-        },
-        resource : 'https://manage.office.com'
-    }));
-
-var Office365Env = {
-    name: 'Office365Management',
-    portalUrl: 'http://go.microsoft.com/fwlink/?LinkId=254433',
-    publishingProfileUrl: 'http://go.microsoft.com/fwlink/?LinkId=254432',
-    managementEndpointUrl: 'https://manage.office.com/',
-    resourceManagerEndpointUrl: 'https://management.azure.com/',
-    sqlManagementEndpointUrl: 'https://management.core.windows.net:8443/',
-    sqlServerHostnameSuffix: '.database.windows.net',
-    galleryEndpointUrl: 'https://gallery.azure.com/',
-    activeDirectoryEndpointUrl: 'https://login.microsoftonline.com/',
-    activeDirectoryResourceId: 'https://manage.office.com',
-    activeDirectoryGraphResourceId: 'https://graph.windows.net/',
-    activeDirectoryGraphApiVersion: '2013-04-05',
-    storageEndpointSuffix: '.core.windows.net',
-    keyVaultDnsSuffix: '.vault.azure.net',
-    azureDataLakeStoreFileSystemEndpointSuffix: 'azuredatalakestore.net',
-    azureDataLakeAnalyticsCatalogAndJobEndpointSuffix: 'azuredatalakeanalytics.net'
-};
-
-var g_appAdCreds = new msRestAzure.ApplicationTokenCredentials(
-        process.env.CUSTOMCONNSTR_APP_CLIENT_ID,
-        process.env.APP_TENANT_ID,
-        process.env.CUSTOMCONNSTR_APP_CLIENT_SECRET,
-        { 'tokenCache': tokenCache,
-          environment : Office365Env 
-        });
+var g_appAdCreds = new ApplicationTokenCredentials(
```

Comment:
I don't this this will work because I can't find where the token resource is set  - `activeDirectoryResourceId: 'https://manage.office.com',`from the env deleted above.
I almost sure the default resource is `https://management.azure.com/`

Reply:
Also could you please check if MS added O365 API support to the new SDK?

Reply:
So it looks like, No they did not. I put the environment back in. i got a bit overzealous with the cutting.